packages/ui/src/lib/Button/docs/Button.example.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { Fragment } from 'react';
+import { Button } from '../Button';
+
+  export const ButtonExample =  ({...args}) => {
+    return (
+        <Fragment>
```

Comment:
why not <>

Reply:
idk, i didn't make this component, just moved from internal lib.